CDCgov/prime-simplereport
# Update copy for printable multiplex results (#3493)

# FRONTEND PULL REQUEST

## Related Issue
Resolves #3493 

Previous PR for this effort here: https://github.com/CDCgov/prime-simplereport/pull/3835
- Design had been previously approved by @kenieh
- Closed due to merge conflicts

## Changes Proposed
This PR should accomplish the following:
- update the printable test result view both in the SimpleReport app and the Patient Experience App to
  - include multiplex results in English and Spanish
  - include guidance based on multiplex results in English and Spanish
- update the header copy and styles of the printable test results page
- update Storybook components to include multiplex results

## Additional Information
N/A

## Testing
**Multiplex Enabled**
_SimpleReport experience_
1. Go to the "Test Results" page
2. See the print view for a variety of combination of test results (multiplex and non multiplex, corrected, duplicated, etc..)
  - check the Spanish and English translations
  - check the guidance shows up correctly
  - check the symbols next to the test results show up correctly (e.g. Positive (+), etc...)
  - ensure when you click "Print" no content is cut off
  - ensure the print view appears as defined in [Figma](https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=6991%3A25519)
(difference from Figma) (Based on feedback from Jayna and Emma, we will ALWAYS display the negative COVID guidance when there is a negative COVID result)
 
_Patient experience_
1. Go to the patient results section of the patient experience
2. Ensure the result still shows up (PXP copy will be changed in [this PR ](https://github.com/CDCgov/prime-simplereport/pull/3924))
3. Click "Download result"
4. Ensure step 2 of the SimpleReport experience works
5. Ensure NPI shows up and the results should appear correctly

**Multiplex Disabled**
Ensure the PXP and Test Results page print downloads continue to work

## Screenshots / Demos
Non multiplex
<img width="506" alt="Screen Shot 2022-06-30 at 5 40 01 PM" src="https://user-images.githubusercontent.com/20211771/176790685-a5b1e916-bb01-4102-bdab-6df088506103.png">

<img width="499" alt="Screen Shot 2022-06-30 at 5 44 40 PM" src="https://user-images.githubusercontent.com/20211771/176791361-61406b47-32f3-4a46-ab84-7496c037dd0e.png">

Multiplex
<img width="658" alt="Screen Shot 2022-06-30 at 5 45 43 PM" src="https://user-images.githubusercontent.com/20211771/176791403-b06dec15-7670-4de0-9439-b4cc3a4308a4.png">
<img width="657" alt="Screen Shot 2022-06-30 at 5 45 51 PM" src="https://user-images.githubusercontent.com/20211771/176791409-dc2a3675-4d8a-4828-ab7c-dd40c91fd2cd.png">

<img width="662" alt="Screen Shot 2022-06-30 at 5 50 02 PM" src="https://user-images.githubusercontent.com/20211771/176791415-93d0551f-51a7-4f08-88aa-ee3809793934.png">
<img width="659" alt="Screen Shot 2022-06-30 at 5 50 07 PM" src="https://user-images.githubusercontent.com/20211771/176791419-a4ce762f-6e08-47c3-be9b-f4fa9f36fc25.png">

## Checklist for Author and Reviewer

### Design

### Content

### Support

### Security
